src/components/booklist/index.jsx

```diff
@@ -0,0 +1,24 @@
+import Taro, { Component } from '@tarojs/taro';
+import { View } from '@tarojs/components';
+import BookListItem from '../booklist-item';
+
+class BookList extends Component {
+
+  //Using global custom styles in app.scss
+  static options = {
+    addGlobalClass: true
+  }
+
+  render() {
+    const { data: books } = this.props;
+    return(
+      <View>
+        {books.map(book =>
+          <BookListItem key={book.id} data={book} />
+        )}
+      </View>
+    )
+  }
+}
```

Comment:
if it's not stateful component(which means it doesn't contain state management in the component), it's preferable to use stateless function instead of react class, therefore we can use hooks and other new React feature in the future:
for example this block of code can be refactored to:
`const BookList = ({ data: books }) => (<View> .....</View>);`

Reply:
https://programmingwithmosh.com/javascript/stateful-stateless-components-react/